### DISCUSSION TITLE
Compression

### DISCUSSION THREAD

**Discussion by harryray33** [staxrip/staxrip#1086] [General]
Hi, this isn't a question about the programme as such, I'm just wondering how you got your high level of 7zip compression?

I noticed from the properties that you have used LZMA2 Delta compression, and (I'm not sure) if the archive is solid. Is LZMA2 Delta a command line compression function?
I use the GUI, and I can't seem to get the same level of compression.

Thanks.

**Comment by Dendraspis** [comment]
Hi,
yeah, just `LZMA2` on `Ultra` with `Solid` blocks.
